packages/lib/modules/lbp/lbp.types.ts

```diff
@@ -10,7 +10,7 @@ export type SaleStructureForm = {
   weightAdjustmentType: 'linear_90_10' | 'linear_90_50' | 'custom'
   customStartWeight: number
   customEndWeight: number
-  userActions: 'buy_and_sell' | 'buy_only'
+  userActions: 'buy_and_sell' | 'only_buy'
```

Comment:
would this change break the radio input?

https://github.com/balancer/frontend-monorepo/blob/f6ab45ff0434f9e81f9151a90af3018a03cea108/packages/lib/modules/lbp/steps/SaleStructureStep.tsx#L533-L534

Reply:
same here

https://github.com/balancer/frontend-monorepo/blob/f6ab45ff0434f9e81f9151a90af3018a03cea108/packages/lib/modules/lbp/modal/LbpSummary.tsx#L100-L101

Reply:
switched to using enums instead of strings
can you pls check it again?